### DISCUSSION TITLE
how to use MA play_media Action

### DISCUSSION THREAD

**Discussion by mac-city** [music-assistant/support#3055] [Q&A] [answered]
I am reading the documentation here... [play media action](https://music-assistant.io/faq/massplaymedia/) but can't find anything like that in my home assistant automations.  Did I miss a previous installation step?  I don't see service at all in automations gui.  Wait, I see that service is now called action, but I still don't have mass play media as an option in the gui.

I have MA installed and seems to work fine and see a plugin provider named home assistant that is enabled.

**Comment by OzGav** [comment, marked as answer]
You don’t have the integration installed then. https://music-assistant.io/integration/installation/

**Reply by mac-city** [reply]
That was the ticket!  Thank you!  I can finally have a wake up to music alarm and all my sonos speakers are synchronized.  When I was using the HA sonos entities, they were always a little out of sync.  I am grateful for all your work to make such a powerful addition to HA!
